## Setup

In [ ]:
import pandas as pd
pd.options.display.float_format = '{:.1f}'.format

## How does the median square footage of houses built in the 1930s compare to houses built in the 1940s?

In [ ]:
house_path = 'YOUR PATH HERE'
house_sales = pd.read_csv(house_path)

In [ ]:
sales_1930s = house_sales[house_sales['yr_built'].between(1930, 1939)]
sales_1930s['sqft_living'].median()

1635.0

In [ ]:
sales_1940s = house_sales[house_sales['yr_built'].between(1940, 1949)]
sales_1940s['sqft_living'].median()

1370.0

**Median square footage of houses built in 1930s: 1,635**

**Median square footage of houses built in 1940s: 1,370**

## How does the total number of UFO sightings in the cities of Miami, Orlando, and Jacksonville (combined) compare to the total number of sightings in Seattle, Portland, and Vancouver combined?

In [ ]:
ufos_path = 'YOUR PATH HERE'
sightings = pd.read_csv(ufos_path)

<ipython-input-13-72006e44cd64>:2: DtypeWarning: Columns (5,9) have mixed types. Specify dtype option on import or set low_memory=False.
  sightings = pd.read_csv(ufos_path)


In [ ]:
moj_sightings = sightings[sightings['city'].isin(['miami', 'orlando', 'jacksonville'])]
len(moj_sightings)

631

In [ ]:
spv_sightings = sightings[sightings['city'].isin(['seattle', 'portland', 'vancouver'])]
len(spv_sightings)

1035

**Total number of sightings in Miami, Orlando, and Jacksonville: 631**

**Total number of sightings in Seattle, Portland, and Vancouver: 1,035**